src/components/ReplayableVideoAsset.vue

```diff
@@ -58,13 +59,13 @@ export default {
     };
   },
   methods: {
-    replay() {
+    async replay() {
       const videoPlayer = this.$refs.asset.$el;
       if (videoPlayer) {
         this.showsReplayButton = false;
         // Start video playback from the beginning.
         videoPlayer.currentTime = 0;
-        videoPlayer.play();
+        await videoPlayer.play();
```

Comment:
For the documented scenarios where it may fail and be rejected, we don’t necessarily have a better alternative to the default behavior right now where the specific exception would be logged to the console, so I don’t think so.